### PR TITLE
Fixed a bug that prevented the throttling settings from displaying for PER_ALERT bucket level trigger actions

### PR DESCRIPTION
**Pull request by AWSHurneyt**
Signed-off-by: AWSHurneyt <hurneyt@amazon.com>

### Description
Manually backporting PR https://github.com/opensearch-project/alerting-dashboards-plugin/pull/328.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/CreateTrigger/components/Action/actions/Message.js [hunk 1]
```diff
@@ -194,6 +195,8 @@ export default function Message(
     _.set(values, actionableAlertsSelectionsPath, actionableAlertsSelections);
   }
 
+  if (!displayThrottlingSettings) _.set(values, `${actionPath}.throttle_enabled`, false);
```
Comment (amsiglan) [inline public/pages/CreateTrigger/components/Action/actions/Message.js:198]: Is `_.set(values, `${actionPath}.throttle_enabled`, false);` set to true by default? or should we set it to true here in the else case?

Comment (AWSHurneyt) [reply public/pages/CreateTrigger/components/Action/actions/Message.js:198]: By default, `throttle_enabled` is [false](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/public/pages/CreateTrigger/utils/constants.js#L49). 

Setting `throttle_enabled` to false here helps ensure throttling settings aren't sent to the backend for action configurations that don't support throttling when the user may have previously been experimenting with configurations that do support such settings.

Comment (amsiglan) [reply public/pages/CreateTrigger/components/Action/actions/Message.js:198]: Aah, so if the user was playing with the setting for PER_ALERT and then goes to PER_EXECUTION then we should omit it in that case, right?

**Review by amsiglan — APPROVED** [review]

File: public/pages/CreateTrigger/components/Action/actions/Message.js (hunk 1 above)
Comment (AWSHurneyt) [reply public/pages/CreateTrigger/components/Action/actions/Message.js:198]: For bucket level monitors, we want to offer `PER_ALERT` actions, but we do not want to offer them for `PER_EXECUTION`; however, the bug prevented throttling configurations for both selections.

**Review by Angie-Zhang — APPROVED** [review]

**Pull request merged**